backend/package.json

```diff
@@ -5,8 +5,8 @@
   "author": "see repository",
   "main": "dist/src/index.js",
   "scripts": {
-    "start": "tsoa spec-and-routes && tsc && node dist/src/index.js",
-    "dev": "concurrently \"nodemon\" \"nodemon -x tsoa spec-and-routes\"",
+    "start": "tsoa spec-and-routes && tsc && node dist/backend/src/index.js",
```

Comment:
Since this package.json is inside the `backend/` shouldn`t the `dist/` also be built inside it?

Reply:
It is, but inside of the `dist` there is another `backend` folder due to some code being inside the `common` folder. When creating the `dist` folder it splits the files into `backend` and `common` folders automatically.

Reply:
So it is `backend/dist/backend` and `backend/dist/common`